rotkehlchen/exchanges/coinbasepro.py

```diff
@@ -388,6 +388,14 @@ def query_online_deposits_withdrawals(
         account_to_currency = self.create_or_return_account_to_currency_map()
         for entry in raw_movements:
             try:
+                # Check if the transaction has been canceled. If so it should be skipped
+                if entry['completed_at'] is None:
```

Comment:
it should always have it, but why not play it safe here and do the following:
```suggestion
                if entry.get('completed_at', None) is None:
```